kairon/events/definitions/analytic_pipeline_handler.py

```
@@ -0,0 +1,222 @@
+        self.bot = bot
+        self.user = user
+
+    def validate(self):
+        """Check if pipeline exists and is active."""
+        callback_config = CallbackConfig.objects(
+            bot=self.bot,
+            name=self.callback_name
+        ).first()
+
+        if not callback_config:
+            raise AppException(f"No callback config found for callback: {self.callback_name}")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)